activerecord/lib/active_record/connection_adapters/postgresql/oid/cidr.rb

```diff
@@ -28,6 +28,11 @@ def serialize(value)
             end
           end
 
+          def changed?(old_value, new_value, _new_value_before_type_cast)
+            old_value.class != new_value.class ||
```

Comment:
I'm wondering if it needs `old_value.class != new_value.class` here. I assume it comes from https://github.com/rails/rails/commit/dbdd68b64b781ce3c3363ffbc223730138c49a33 , Cidr `cast_value` method looks like taking care of it https://github.com/rails/rails/blob/734719d8af033062cb7088a1353be6276659e3c1/activerecord/lib/active_record/connection_adapters/postgresql/oid/cidr.rb#L31-L43

Reply:
@yahonda Sure I think too this is not necessary.
I have removed `old_value.class != new_value.class`.